src/deployment/deploy.py

```diff
@@ -306,6 +306,8 @@ def setup_rbac(self):
 
         (password_id, password) = self.create_password(app.object_id)
 
+        logger.info('service principal password: %s', password)
```

Comment:
This needs to include the app ID as well as the password.  Note, this is already being done later in this function, but at the 'debug' level.  

Can you add the command line option "--log_service_principal", and use existing logging at the end of the function at the 'info' level, and 'debug' if the command line is not set?
 
https://github.com/microsoft/onefuzz/blob/10267a3cb4dfd71895aaeb917110b7abecdd4503/src/deployment/deploy.py#L330-L333

Reply:
Accepted and updated my code in the PR